modules/backing-up-red-hat-quay.adoc

```diff
@@ -0,0 +1,251 @@
+      …
+----
++
+[NOTE]
+====
+If you are using a version older than 3.7.0, then you must manually scale down the Operator and manually scale down the Quay deployment. For example, you can run `$ oc scale --replicas=0 deployment quay-operator.v3.6.2 -n openshift-operators` to scale down the Operator, and `$ oc scale --replicas=0 deployment QUAY_MAIN_APP_DEPLOYMENT` to scale down the main application deployment.
+====
+
+. Wait for the `registry-quay-app` pods to disappear. You can check their status by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n <quay-namespace>
+----
++
+Example output:
++
+[source,terminal]
+----
+registry-quay-config-editor-77847fc4f5-nsbbv   1/1     Running            0          9m1s
+registry-quay-database-66969cd859-n2ssm        1/1     Running            0          6d1h
+registry-quay-mirror-758fc68ff7-5wxlp          1/1     Running            0          8m29s
+registry-quay-mirror-758fc68ff7-lbl82          1/1     Running            0          8m29s
+registry-quay-redis-7cc5f6c977-956g8           1/1     Running            0          5d21h
+----
+
+. Identify the Quay PostgreSQL pod name:
++
+[source,terminal]
+----
+$ oc get pod -l quay-component=postgres -n <quay-namespace> -o jsonpath='{.items[0].metadata.name}'
+quayregistry-quay-database-59f54bb7-58xs7
+----
+
+. Obtain the Quay database name:
++
+[source,terminal]
+----
+$ oc -n <quay-namespace> rsh $(oc get pod -l app=quay -o NAME -n <quay-namespace> |head -n 1) cat /conf/stack/config.yaml|awk -F"/" '/^DB_URI/ {print $4}'
+quayregistry-quay-database
+----
+
+. Download a backup database:
++
+[source,terminal]
+----
+$ oc exec quayregistry-quay-database-59f54bb7-58xs7 -- /usr/bin/pg_dump -C quayregistry-quay-database  > backup.sql
+----
+
+. Create and download a backup database:
++
+----
+$ oc exec example-registry-quay-database-66969cd859-n2ssm -- /usr/bin/pg_dump -C example-registry-quay-database > backup.sql
+----
++
+The backup of your Quay database will be saved in the `backup.sql` file. If you are using an external database, refer to Migrating a standalone Quay deployment to a {productname} Operator managed deployment.
+
+. Decode and export the `AWS_ACCESS_KEY_ID`:
++
+[source,terminal]
+----
+$ export AWS_ACCESS_KEY_ID=$(oc get secret -l app=noobaa -n quay-enterprise  -o jsonpath='{.items[0].data.AWS_ACCESS_KEY_ID}' |base64 -d)
+----
+
+. Decode and export the `AWS_SECRET_ACCESS_KEY_ID`:
++
+[source,terminal]
+----
+$ export AWS_SECRET_ACCESS_KEY=$(oc get secret -l app=noobaa -n quay-enterprise -o jsonpath='{.items[0].data.AWS_SECRET_ACCESS_KEY}' |base64 -d)
+----
+
+. Create a new directory and copy all blobs to it:
++
+[source,terminal]
+----
+$ mkdir blobs
+----
++
+[source,terminal]
++
+----
+$ aws s3 sync --no-verify-ssl --endpoint https://$(oc get route s3 -n openshift-storage  -o jsonpath='{.spec.host}')  s3://$(oc get cm -l app=noobaa -n quay-enterprise -o jsonpath='{.items[0].data.BUCKET_NAME}') ./blobs
+----
```

Comment:
For consistency, do we need to replace `quay-enterprise` by `<quay-namespace>`? 
such as :
```
export AWS_ACCESS_KEY_ID=$(oc get secret -l app=noobaa -n <quay-namespace>  -o jsonpath='{.items[0].data.AWS_ACCESS_KEY_ID}' |base64 -d)

export AWS_SECRET_ACCESS_KEY=$(oc get secret -l app=noobaa -n <quay-namespace> -o jsonpath='{.items[0].data.AWS_SECRET_ACCESS_KEY}' |base64 -d)

aws s3 sync --no-verify-ssl --endpoint https://$(oc get route s3 -n openshift-storage  -o jsonpath='{.spec.host}')  s3://$(oc get cm -l app=noobaa -n <quay-namespace> -o jsonpath='{.items[0].data.BUCKET_NAME}') ./blobs

```